common/decrees/00_decree.txt

```diff
@@ -467,7 +463,8 @@ decree_encourage_manufacturing_industry = {
 	#	}	
 	modifier = {
 		building_group_bg_manufacturing_throughput_add = 0.2
-		state_construction_mult = 0.05
+		country_engineers_pol_str_mult = 0.01
+		country_machinists_pol_str_mult = 0.01
```

Comment:
This 1% is too insignificant to add in my opinion, I would suggest making it relevent (like 5%) or removing it. To prevent cluttering.

Reply:
The problem is that its country-wide, else I would. I couldn't determine a state modifier that worked for this purpose. if I made it 5% then with 10 of these those jobs are +50% across the country and suddenly you have a big trade unions for 'free'.

Reply:
ah that is certainly a bit awkward, but interesting. Lets see how this goes.